src/grading-settings/GradingSettings.jsx

```diff
@@ -153,16 +221,18 @@ const GradingSettings = ({ intl, courseId }) => {
           data-testid="grading-settings-save-alert"
           role="dialog"
           actions={[
-            <Button
-              variant="tertiary"
-              onClick={handleResetGradingCutoffs}
-            >
-              {intl.formatMessage(messages.buttonCancelText)}
-            </Button>,
-            <Button onClick={handleSendGradingSettingsData}>
-              {intl.formatMessage(messages.buttonSaveText)}
-            </Button>,
-          ]}
+            !isQueryPending && (
+              <Button variant="tertiary" onClick={handleResetPageData}>
+                {intl.formatMessage(messages.buttonCancelText)}
+              </Button>
+            ),
+            <StatefulButton
+              key="statefulBtn"
+              onClick={handleSendGradingSettingsData}
+              state={isQueryPending ? RequestStatus.PENDING : 'default'}
```

Comment:
does it make sense to add `default` to state constants?

Reply:
yeah! let's move to `src/constants.js` it and keep 'default' and 'pending' there

Reply:
@sendr a string with a value of "default" is provided by the default Paragon for this component. I think it's a good idea to follow this principle. 
@ruzniaievdm the file with constants has been created, thanks

Reply:
let's store it as

```
export const STATEFUL_BUTTON_STATES = {
  pending: 'pending',
  default: 'default',
};
```